assets/src/blocks/PostsList/index.js

```diff
@@ -22,7 +22,6 @@ export const registerPostsListBlock = () => {
     title: 'Posts List',
     icon: 'list-view',
     description: __('Insert a list or grid of the latest articles, press releases, and/or publications, organized by publication date. ', 'planet4-blocks-backend'),
-    category: 'planet4-blocks-beta',
```

Comment:
Don't we need to use `category: 'planet4-blocks'` instead?

Reply:
You're absolutely right, fixed!